src/components/GraphWorkshop/ParametersPanel/ParametersPanel.tsx

```diff
@@ -17,7 +18,7 @@ import './ParametersPanel.scss';
 const ParametersPanel = () => {
 	const dataSources = useRecoilValue(dataSourcesAtom);
 	const [dataSource, setDataSource] = useState<OptionItem>(null);
-	const [graphType, setGraphType] = useState<OptionItem>(null); //for the time being all of these are useState
+	const [graphType, setGraphType] = useState<OptionItem<GraphType>>(null); //for the time being all of these are useState

```

Comment:
I don't know which one of us will merge before the other one, just make sure the word cloud works fine both in creation mode and edit mode

Reply:
I'll wait for your merge first and I make it editable